python/cudf/cudf/_lib/pylibcudf/hashing.pyx

```diff
@@ -0,0 +1,258 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+from libc.stdint cimport uint32_t, uint64_t
+from libcpp.memory cimport unique_ptr
+from libcpp.utility cimport move
+
+from cudf._lib.pylibcudf.libcudf.column.column cimport column
+from cudf._lib.pylibcudf.libcudf.hash cimport (
+    DEFAULT_HASH_SEED,
+    md5 as cpp_md5,
+    murmurhash3_x64_128 as cpp_murmurhash3_x64_128,
+    murmurhash3_x86_32 as cpp_murmurhash3_x86_32,
+    sha1 as cpp_sha1,
+    sha224 as cpp_sha224,
+    sha256 as cpp_sha256,
+    sha384 as cpp_sha384,
+    sha512 as cpp_sha512,
+    xxhash_64 as cpp_xxhash_64,
+)
+from cudf._lib.pylibcudf.libcudf.table.table cimport table
+
+from .column cimport Column
+from .table cimport Table
+
+LIBCUDF_DEFAULT_HASH_SEED = DEFAULT_HASH_SEED
+
+cpdef Column murmurhash3_x86_32(
+    Table input,
+    uint32_t seed=DEFAULT_HASH_SEED
+):
+    """Computes the MurmurHash3 32-bit hash value of each row in the given table.
+
+    For details, see :cpp:func:`murmurhash3_x86_32`.
+
+    Parameters
+    ----------
+
```

Comment:
I don't know if we can enforce pydocstyle on Cython files, but we should try. We don't need these empty lines. Please apply this fix throughout.
```suggestion
```